notready/MNL000032.yaml

```diff
@@ -1,32 +1,56 @@
 bioguide: MNL000032
 contact_form:
-  method: post
-  action:
+  driver: chrome
```

Comment:
Since this is an only one in the "not ready" folder, does it need to be updated?